plugins/azure/frontdoor/frontDoorManagedIdentity.js

```diff
@@ -0,0 +1,60 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Front Door Managed Identity Enabled',
+    category: 'Front Door',
+    domain: 'Content Delivery',
+    severity: 'Medium',
+    description: 'Ensures that Azure Front Door have managed identity enabled.',
```

Comment:
can we write it as description: 'Ensures that Front Door profiles have managed identity enabled.',
Comment